### DISCUSSION TITLE
q w e r shortcut key problem

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1208] [General]
Uploading dd0ea892bb3d1425ef6601bcd1106956.mp4…

**Comment by Terracotta-62** [comment]


**Reply by MAKOMO** [reply]
Not sure what I am seeing here and what the problem is. Could you add an explanation?

Note that those shortcuts work by pressing the corresponding shortcut key, followed by the value to be set for that event by pressing number keys. I don't see here that you press any number keys after the shortcut key.

**Comment by Terracotta-62** [comment]
Enter two digits after the shortcut letter key. Arrow key input values are not accepted.
Thank you for your answer
